### PR TITLE
Icicle chart

### PR DESCRIPTION
**Pull request by milanzmitrovic**

### CODE REVIEW

**Comment by someshfengde** [comment]
great apps @milanzmitrovic  :smile:

**Comment by milanzmitrovic** [comment]
@someshfengde Thanks Somesh!

**Comment by AnnMarieW** [comment]
@milanzmitrovic I agree nice apps!

For the app with tabs, what do you think about removing the close button.  It makes it so that when you make selections, you can still see the figure.  Plus, it's not necessary since you are using the modal header which has a close button built into it.  

Without button:
image

With button:

image

File: examples/modal_treemap.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+
+import pandas as pd
+from dash import Dash, Input, Output, callback, dcc, html, State
+import plotly.express as px
+import dash_bootstrap_components as dbc
+from datetime import date
```
Comment (AnnMarieW) [inline examples/modal_treemap.py:6]: Can remove unused import

**Comment by milanzmitrovic** [comment]
Hi Ann,

It makes sense. Thanks for suggestion.

I am updating PR now.

**Comment by milanzmitrovic** [comment]
Here is link with updated PR.
https://github.com/AnnMarieW/dash-app-gallery/pull/84/files

It seems like it has my last updates. Please let me know if it works? I am not best experienced with git, sometimes I make mess with creating PR :)

**Comment by Coding-with-Adam** [comment]
icicle chart added through a new PR

**Pull request closed without merging**